src/libsyncengine/progress/syncfileitem.cpp

```diff
@@ -18,15 +18,12 @@
 
 #include "syncfileitem.h"
 
-#include <ctime>
-
 namespace KDC {
 
 SyncFileItem::SyncFileItem() :
-    _type(NodeType::Unknown), _path(SyncPath()), _newPath(std::nullopt), _localNodeId(std::nullopt), _remoteNodeId(std::nullopt),
+    _type(NodeType::Unknown), _newPath(std::nullopt), _localNodeId(std::nullopt), _remoteNodeId(std::nullopt),
     _direction(SyncDirection::Unknown), _instruction(SyncFileInstruction::None), _status(SyncFileStatus::Unknown),
-    _conflict(ConflictType::None), _inconsistency(InconsistencyType::None), _size(0), _dehydrated(false), _confirmed(false),
-    _timestamp(std::time(0)) {}
+    _conflict(ConflictType::None), _inconsistency(InconsistencyType::None), _size(0) {}
```

Comment:
Those variables could be initialized in header file

Reply:
Yes, done.